main.py

```
@@ -1,8 +1,12 @@
 import argparse
+import hashlib
 import logging
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)